binding/dotnet/PorcupineTest/MainTest.cs

```diff
@@ -33,28 +64,48 @@ public static void ClassInitialize(TestContext testContext)
             }
         }
 
-        [TestMethod]
-        public void TestFrameLength()
+        private static string appendLanguage(string s, string language)
```

Comment:
I know this is annoying, but in C# all function names start with an uppercase, regardless of public/private status
```suggestion
        private static string AppendLanguage(string s, string language)
```